From 790fae67e5852b2758cbd8a5a80a3303753fe138 Mon Sep 17 00:00:00 2001
From: unknown author <cooker@mandrivalinux.org>
Date: Thu, 11 Feb 2010 14:38:58 +0000
Subject: [PATCH 34/36] mips define isa macros

Don't know if the change is breaking something but it's a little
bit better than no value at all. I'm not even sure of how to handle
n32 :(
---
 installplatform |    8 	8 +	0 -	0 !
 1 file changed, 8 insertions(+)
---
 installplatform |    8 ++++++++
 1 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/installplatform b/installplatform
index bf10732..dcf338c 100755
--- a/installplatform
+++ b/installplatform
@@ -106,6 +106,14 @@ for ARCH in noarch `grep ^arch_canon $RP
 	CANONCOLOR=0
 	FILTER="grep -v -E '^(%optflag|%__isa)'"
 	;;
+    mipsel)
+        ISANAME=mipsel
+	ISABITS=32
+	;;
+    mips)
+        ISANAME=mips
+	ISABITS=32
+	;;
   esac
 
   # skip architectures for which we dont have full config parameters
-- 
1.6.4.4

